sources/packages/web/src/services/http/UserApi.ts

```diff
@@ -20,7 +20,7 @@ export class UserApi extends HttpBaseClient {
     }
   }
 
-  public async bceidAccount(headers?: any): Promise<BCeIDDetailsDto | null> {
+  public async bceidAccount(headers?: any): Promise<BCeIDDetailsDto> {
```

Comment:
I noticed that this method is also returning "null" from the API side. Is there a reason to change it in the Vue application?